lessons/lesson_6/Lesson_6_Home_Work_conditions.py

```diff
@@ -1,3 +1,28 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+import time
+
+driver = webdriver.Chrome()
+driver.get("https://portnov_admin-trials711.orangehrmlive.com/client/#/dashboard")
+
+driver.find_element(By.CSS_SELECTOR, "input[id='txtUsername']").send_keys("Admin")
+driver.find_element(By.CSS_SELECTOR, "input[id='txtPassword']").send_keys("qTJn5@5APu")
+driver.find_element(By.CSS_SELECTOR, "button[type='submit']").click()
+
+time.sleep(5)
+driver.find_element(By.XPATH, "//li[@id='left_menu_item_10']").click()
+time.sleep(5)
+driver.find_element(By.XPATH, "//div[@data-tooltip='Add User']/a").click()
+time.sleep(5)
+
+
+driver.find_element(By.XPATH, "//label[contains(text(),'Disabled')]").click()
```

Comment:
good, but you are clicking before checking if its's enabled or disabled. So the element will be clicked anyway. the correct implementation is: enabled_button = driver.find_element(By.XPATH, '//label[text()="Enabled"]/..')
disabled_button = driver.find_element(By.XPATH, '//label[text()="Disabled"]/..')

if 'selected-option' in enabled_button.get_attribute('class'):
    disabled_button.click()
    time.sleep(2)
    driver.find_element(By.XPATH, '//button[contains(text(),"×")]').click()
    time.sleep(5)